ruby/ql/lib/codeql/ruby/frameworks/core/Array.qll

```diff
@@ -266,11 +271,11 @@
~          rl.isInclusive() and end = e
           or
           rl.isExclusive() and end = e - 1
         ) and
-        this = "[](" + start + ".." + end + ")"
+        this = methodName + "(" + start + ".." + end + ")"
```

Comment:
This means that we are now synthesizing two methods, whereas before there was just one. But that should be OK.

Reply:
Right, I forgot to mention it but I thought I should make the change for consistency.